algorithms/gr-dij/java/Dijkstra.java

```diff
@@ -0,0 +1,89 @@
+import java.lang.*;
```

Comment:
Codacy found an issue: [Avoid importing anything from the package java.lang](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6333211)